python/dta/density.py

```
@@ -369,38 +280,17 @@ def _package_counts(unrolled_data, grid_dims):
         3d array of counts in [time, r, theta] format.
 
     """
-    nframes = grid_dims.Nframes
+    nframes = int(round(unrolled_data.shape[0] / grid.r.n_bins))
```

Ruff (0.15.18)

[warning] Value being cast to `int` is already an integer

Remove unnecessary `int` call

(RUF046)